src/_scss/pages/search/searchSidebar.scss

```diff
@@ -3,16 +3,62 @@
     @import '../../elements/_inputs';
     @import '../../elements/_labels';
 
+    .filter-header {
+        font-size: $base-font-size;
+        font-weight: bold;
+        padding-bottom: 1rem;
+        padding-left: 1rem;
+        display: inline-block;
+    }
+
 	.search-option {
 		margin-bottom:1rem;
 
         label {
             @include usa-label;
         }
-	}
 
-	.search-option b::after {
-		content: ' \02193';
+        svg {
```

Comment:
Since this is for a specific component (the filter expand/collapse button), this should be in its own file.